contracts/common/bases/PortfolioBaseUpgradeable.sol

```diff
@@ -645,31 +652,25 @@ abstract contract PortfolioBaseUpgradeable is
 
         // no prior investment into any of the strategies by the portfolio
         if (totalEquity == 0) {
-            return calculateEmbeddedFee(FeeType.Withdrawal, params);
+            return calculateEmbeddedFeeTargetAllocation(feeType, params);
         }
         // there is at least one strategy with investment by the portfolio
         else {
             for (uint256 i = 0; i < investableDescsLength; i++) {
                 IInvestable embeddedInvestable = investableDescs[i].investable;
                 embeddedFee +=
-                    (uint256(embeddedInvestable.getTotalWithdrawalFee(params)) *
-                        currentInvestableEquities[i]) /
+                    (uint256(
+                        (feeType == FeeType.Withdrawal)
+                            ? embeddedInvestable.getTotalWithdrawalFee(params)
+                            : embeddedInvestable.getTotalManagementFee(params)
```

Comment:
I think this will give wrong value to performance fee

Reply:
calculateEmbeddedFeeTargetAllocation is not called for performance fees

Currently what the getTotalPerformanceFee returns is the weighted average of the embedded investables. This is not ideal, but to do the right thing: return 'the actual performance fee in percentage that would be taken' if we called the takePerformanceFee on the portfolio would require us to add some new methods to the embedded investables.

Let's postpone doing the ideal thing until anyone choses to use this (people don't even call the more common getWithdrawalFee method.